spec/yaml/swiftnav/sbp/ssr.yaml

```
@@ -182,8 +182,9 @@ definitions:
         type: u8
         units: mm
         desc: >
-          Modified DF389. class 3 MSB, value 5 LSB.
-          stddev = (3^class * (1 + value/16) - 1)
+          Encoded standard deviation leaning on RTCM DF389 specification.
```

Question: not sure about the English wording here. Would the following not be clearer?
```suggestion
          Encoded standard deviation similar to RTCM DF389 specification.
```